docs/content/parsers/regexp.md

```diff
@@ -20,13 +20,14 @@ function regexp(re: RegExp, expected: string): Parser<string>
 
 The regular expression must obey two simple rules:
 
-- It *does* use `g` flag. Flags like `u` and `i` are allowed and can be added if needed.
+- It automatically injects `g` flag if it's *not* present so the parser will work correctly. 
```

Comment:
Tbh this line doesn't really correspond to the above one, because it's not a "rule", just a note about the implementation.

I'd just add a [warning container](https://vitepress.vuejs.org/guide/markdown#custom-containers) in the "Description" right before the text that mentions this behavior. I could commit that myself.